fe/fe-core/src/main/java/com/starrocks/backup/RestoreJob.java

```diff
@@ -509,14 +510,27 @@ private void checkAndPrepareMeta() {
                         return;
                     }
                     LOG.debug("get intersect part names: {}, job: {}", intersectPartNames, this);
-                    if (localOlapTbl.getSignature(BackupHandler.SIGNATURE_VERSION, intersectPartNames)
-                            != remoteOlapTbl.getSignature(BackupHandler.SIGNATURE_VERSION, intersectPartNames)) {
+                    if (localOlapTbl.getSignature(BackupHandler.SIGNATURE_VERSION, intersectPartNames, true)
```

Comment:
If there are some columns with different columns, is the data accessible by storage engine after restore?